tvtk/special_gen.py

```diff
@@ -319,8 +319,11 @@ def __iter__(self):
                 yield obj.GetPoint(i)
 
         def _check_key(self, key, n):
-            if type(key) != type(1):
-                raise TypeError, "Only integers are valid keys."
+            ##############################################
+            # Allow int and long keys. Fixes GH Issue 173.
+            ##############################################
+            if type(key) not in [type(1), type(1L)]:
```

Comment:
@prabhuramachandran Can you tell why the `_check_key` calls have been added? Was that necessary for VTK < 5.10? Is it safe to drop these calls, as they are also used for IdList and DataArray?